### ISSUE TITLE
Arm docker version

### ISSUE THREAD

**Issue by techkek** [lamarios/monopoly#21]
is it possible to have a docker version for arm? arm-64 or arm-32 is indifferent. (i want to run it on raspberry pi)
Thank you!

**Comment by lamarios** [comment]
I'll look into it but I have no way of testing it.

**Comment by techkek** [comment]
I created a very simple docker-compose.yml for the game part (using your Dockerfile) and it works perfectly on arm.
This is the docker-compose.yml:
`version: '3.3'
services:
    monopoly-game:
        container_name: monopoly-game
        build:
            context: .
        environment:
          - HTTP=true
        ports:
          - 8442:8443
        restart: unless-stopped
`
I also translated the game entirely into Italian, if you are interested I can also publish it (obviously saying who did it).
I can close the issue because I solved it.
